plugins/vela.yaml

```diff
@@ -3,14 +3,14 @@ kind: Plugin
 metadata:
   name: vela
 spec:
-  version: "v1.1.13"
+  version: "v1.2.0"
   platforms:
   - selector:
       matchLabels:
         os: linux
         arch: amd64
-    uri: https://github.com/oam-dev/kubevela/releases/download/v1.1.13/kubectl-vela-v1.1.13-linux-amd64.tar.gz
-    sha256: 54818cb474cfd11264403fbe77f7a32dbd5a340aec074c0edcb3bb8e91e83960
+    uri: https://github.com/oam-dev/kubevela/releases/download/v1.2.0/kubectl-vela-v1.2.0-linux-amd64.tar.gz
+    sha256: 8815c132c524e7207d4d15da8e511b311a4d1b16609150aaa3cbea465c0ac669
     files:
     - from: "*/kubectl-vela"
```

Comment:
@wonderflow please send a manual PR. It seems this path is no longer present in the archive. 
/close